.mlc_config.json

```diff
@@ -0,0 +1,29 @@
+{
+  "aliveStatusCodes": [
+    403,
+    200,
+    206
```

Comment:
If we receive any of these HTTP codes, treat them as a sign that the links are alive.